org.eclipse.wildwebdeveloper/plugin.xml

```diff
@@ -545,11 +545,25 @@
                   <iterate>
                      <adapt
                            type="org.eclipse.core.resources.IResource">
+                        <test
+                              forcePluginActivation="true"
+                              property="org.eclipse.wildwebdeveloper.isHTMLProject">
```

Comment:
At the moment, I don't think the launch shortcuts allow to properly handle projects, but work fine with files.
So I'd suggest that instead of checking for HTML project, we currently only check for HTML files. We can add projects in a separate commit,

Reply:
I agree. I'll make it check if the user is trying to launch a HTML file rather than checking if the file belongs to a project that contains a `index.html`.